providers/netsuite.go

```diff
@@ -1,8 +1,16 @@
 package providers
 
-const Netsuite Provider = "netsuite"
+const (
+	Netsuite Provider = "netsuite"
 
-// nolint:lll
+	// NetsuiteModuleSuiteQL is a read-only module that uses SuiteQL to read data.
+	NetsuiteModuleSuiteQL = "suiteql"
+
+	// NetsuiteModuleRESTAPI is a read-write module that uses the REST API to read and write data.
+	NetsuiteModuleRESTAPI = "restapi"
```

Comment:
what do you think of `rest` as the module name instead? it's simpler and reads better than `restapi`, which looks like it should be pronounced "res-tah-pee"

Reply:
similarly the constant name can be `NetsuiteModuleREST` for brevity